src/amp-story-player/amp-story-player-impl.js

```diff
@@ -843,14 +768,14 @@ export class AmpStoryPlayer {
 
   /** Sends a message muting the current story. */
   mute() {
-    const {iframeIdx} = this.stories_[this.currentIdx_];
-    this.updateMutedState_(iframeIdx, true);
+    const story = this.stories_[this.currentIdx_];
```

Comment:
Pretty nitty but if it's easy I think it'd be cool to pass the StoryDef as a parameter. Right now for ex, you couldn't mute the current story and unmute the next one. I know it wouldn't work on the web anyway but it would be nice to support it eventually, and to have method signature consistency.

Reply:
Since this is a public API it would mean a breaking change :/